tests/job/test_genericJob.py

```diff
@@ -502,5 +504,56 @@ def test_return_codes(self):
             pass
         self.assertTrue(j.status.aborted, "Job did not abort even though return code is 2!")
 
+    def test_job_executor_set(self):
```

Comment:
I guess this test can be moved over to the `Server` tests since the fact it lives on a job is an uneccesary complication.